kubetest/main.go

```diff
@@ -168,7 +169,8 @@ func defineFlags() *options {
 	flag.StringVar(&o.nodeTestArgs, "node-test-args", "", "Test args specifically for node e2e tests.")
 	flag.BoolVar(&o.noAllowDup, "no-allow-dup", false, "if set --allow-dup will not be passed to push-build and --stage will error if the build already exists on the gcs path")
 	flag.BoolVar(&o.nodeTests, "node-tests", false, "If true, run node-e2e tests.")
-	flag.BoolVar(&o.perfTests, "perf-tests", false, "If true, run tests from perf-tests repo.")
+	flag.BoolVar(&o.perfTests, "perf-tests", false, "If true, run tests from perf-tests repo using run-e2e.sh from the main directory.")
```

Comment:
Could you change this to a string flag `--perf-test` which takes argument as the test to be run (i.e cluster-loader, benchmark, etc)? And make --perf-tests-args actually take arguments for that test?

Reply:
done.